README.md

```diff
@@ -1,25 +1,23 @@
-<img src="https://raw.githubusercontent.com/microsoft/torchgeo/main/logo/logo-color.svg" width="400" alt="TorchGeo"/>
+<img src="https://raw.githubusercontent.com/microsoft/torchgeo/main/logo/logo-color.svg" width="400" alt="TorchGeo logo"/>
 
-TorchGeo is a [PyTorch](https://pytorch.org/) domain library, similar to [torchvision](https://pytorch.org/vision), that provides datasets, transforms, samplers, and pre-trained models specific to geospatial data.
+TorchGeo is a [PyTorch](https://pytorch.org/) domain library, similar to [torchvision](https://pytorch.org/vision), providing datasets, samplers, transforms, and pre-trained models specific to geospatial data.
 
 The goal of this library is to make it simple:
 
-1. for machine learning experts to use geospatial data in their workflows, and
-2. for remote sensing experts to use their data in machine learning workflows.
+1. for machine learning experts to work with geospatial data, and
+2. for remote sensing experts to explore machine learning solutions.
 
-See our [installation instructions](#installation), [documentation](#documentation), and [examples](#example-usage) to learn how to use TorchGeo.
```

Comment:
These links worked on GitHub but not on PyPI. We _could_ fix them, but I'm not sure how useful they actually are since these 3 sections immediately follow the links.